test/jit/test_freezing.py

```diff
@@ -1927,6 +1927,30 @@ def test_pool3d_batchnorm(self):
                 FileCheck().check("aten::to_dense").check_next("return").run(mod.graph)
                 self.assertTrue(torch.allclose(sub_model(inp), mod(inp)))
 
+    @unittest.skipIf(not torch._C.has_mkldnn, "MKL-DNN build is disabled")
+    @skipIfNoTorchVision
+    def test_layernorm(self):
+        with set_default_dtype(torch.float):
+            model = torchvision.models.resnet18()
```

Comment:
idk if we should be instantiating resnet just to get its submodules... but i guess im guilty of that too lol